config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/beans/ScheduledTask.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2018. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Map;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonInclude(NON_NULL)
+public class ScheduledTask {
+    private String policy;
+    private boolean isOneNode;
+    private String jobType;
+    private String jobStatus;
+    private String executionDate;
+    private String cronExpression;
+    private boolean shouldExecuteOnCreate;
```

Comment:
do we need this property in this scenario? 
This makes sense while creating by policy manager that you want to run the task right after creation, but in case of ephemeral gateways, I am not sure if makes sense to run something right after startup (and I don't even know if this will happen).

Reply:
If this does get run right after startup then it could be valuable, but if it doesn't I agree that it isn't very useful.

Reply:
I dug up the documentation here: https://wiki.l7tech.com/mediawiki/index.php/Scheduled_Task

So when shouldExecuteOnCreate is true, The background task will run immediately after Gateway startup (with default of 16000 millisecond delay), and then continue to run in recurring mode, or on an execution date depending on the jobType.

If shouldExecuteOnCreate is false, the background task would run in recurring mode, or on an execution date depending on the jobType.